pkg/cvo/sync_worker.go

```diff
@@ -213,7 +213,13 @@ func (w *SyncWorker) Update(generation int64, desired configv1.Update, overrides
 		w.work.Generation = generation
 	}
 
-	if work.Empty() || equalSyncWork(w.work, work) {
+	if work.Empty() {
+		klog.V(5).Info("Update work is empty; ignoring requested change")
```

Comment:
`Update work is empty` is confusing to me. Can you expand this a little?

Reply:
What about `Update work is empty i.e. there is no version to update`

Reply:
Updated to `Update work has no release image; ignoring requested change` with 0802f07fb0 -> 48b59d1414.